CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ This section is for maintaining a changelog for all breaking changes for the cli
 
 ### Fixed
 - Fix version and build ([#254](https://github.com/opensearch-project/opensearch-java/pull/254))
+- Fix queries not preserving boost and name when converted to builders ([#1181](https://github.com/opensearch-project/opensearch-java/pull/1181))
```

Comment:
Should I be including this in the changelog here?

Reply:
Yes, thank you!